ci/container/change_snowflake_test_pwd.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
```

Comment:
This file seems to be the same for all three connectors do you think it could just have been committed to mono repo?

Reply:
I feel like this leaves some room for each driver to fully control this kind of things. And given that the test password used by each driver is configured by driver themselves, I would suggest to leave the script with them as well.